crates/forge/tests/e2e/io_operations.rs

```diff
@@ -31,7 +31,7 @@ fn file_reading() {
         [FAIL] file_reading_integrationtest::test::non_existent
         
         Failure data:
-            "No such file or directory [..]"
+            "[..] file [..]"
```

Comment:
Could we make this failure message dependent on platform (applies to other places)?

Reply:
Actually, I'd do the opposite. We should wrap these errors so we don't output OS errors directly to the user. And specifically, we shouldn't assert specific os errors in tests.

Reply:
But until we do so, maybe I can assert the os specific error here

Reply:
Changed all of them to be OS dependent except for one assert that asserted the path `scripts/`.